app/services/evaluation/OnlineTestPassmarkRulesEngine.scala

```diff
@@ -53,12 +53,13 @@ object OnlineTestPassmarkRulesEngine extends OnlineTestPassmarkRulesEngine {
   private def evaluateScore(candidateScores: CandidateScoresWithPreferencesAndPassmarkSettings)(schemeName: String) = {
     val passmark = candidateScores.passmarkSettings.schemes.find(_.schemeName == schemeName)
       .getOrElse(throw new IllegalStateException(s"schemeName=$schemeName is not set in Passmark settings"))
```

Comment:
I won't use this class in phase1 evaluation. If it causes you any problems in refactoring feel free to remove it and test